policy/modules/system/systemd.te

```diff
@@ -1032,6 +1032,10 @@ modutils_read_module_deps(systemd_modules_load_t)
 
 systemd_log_parse_environment(systemd_modules_load_t)
 
+selinux_getattr_fs(systemd_modules_load_t)
+
+kernel_getattr_proc(systemd_modules_load_t)
```

Comment:
We have a `kernel_dontaudit_getattr_proc()` call above. Is it breaking without this access?